content/en/integrations/journald.md

```diff
@@ -51,8 +51,11 @@ Then add this configuration block to your `journald.d/conf.yaml` file to start c
 ```
 logs:
   - type: journald
+    container_mode: true
 ```
 
+**Note**: `container_mode` is set to `true`, to automatically set the `source` attribute of your logs to the corresponding short image name of the container instead of simply `docker`.
```

Comment:
```suggestion
**Note**: With Agent 7.17+ if `container_mode` is set to `true`, the `source` attribute of your logs is automatically set to the corresponding short image name of the container instead of simply `docker`.
```